src/aks-preview/azext_aks_preview/tests/latest/recordings/test_aks_abort.yaml

```diff
@@ -16,7 +16,7 @@ interactions:
       - AZURECLI/2.53.0 azsdk-python-azure-mgmt-containerservice/28.0.0b Python/3.8.10
         (Linux-5.15.0-1051-azure-x86_64-with-glibc2.29)
     method: GET
-    uri: https://management.azure.com/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/clitest000001/providers/Microsoft.ContainerService/managedClusters/cliakstest000002?api-version=2025-02-02-preview
+    uri: https://management.azure.com/subscriptions/00000000-0000-0000-0000-000000000000/resourceGroups/clitest000001/providers/Microsoft.ContainerService/managedClusters/cliakstest000002?api-version=2025-03-02-preview
   response:
```

Comment:
should we remove the property `enablePodSecurityPolicy` from the recording files?

Reply:
agree, please create a new PR to update the recording files